rs/registry/canister/src/registry_lifecycle.rs

```diff
@@ -58,6 +66,88 @@ pub fn canister_post_upgrade(
     }
 }
 
+/*
+Three node operators had corrupted NodeOperatorRecords which caused issues during migration (https://dashboard.internetcomputer.org/proposal/139210).
+
+1- record - node_operator_k: 3nu7r node_operator_v: ujq4k
+   As result:
+   * 3nu7r missed the update to max_rewardable_nodes during migration
+   * ujq4k got ovewritten by 3nu7r value.
```

Comment:
This seems to be a defect with the record of ujq4k, not 3nu7r, and so this defect should be mentioned under ujq4k, not 3nu7r.